rdt/transformers/__init__.py

```diff
@@ -180,3 +180,18 @@ def get_default_transformer(sdtype):
     """
     default_transformers = get_default_transformers()
     return default_transformers[sdtype]
+
+
+@lru_cache()
+def get_generator_transformers():
+    """Get transformers that generate data.
+
+    Returns:
+        tuple of transformers that generate data.
+    """
+    generators = []
+    for _, transformer in TRANSFORMERS.items():
+        if not transformer.OUTPUT_SDTYPES:
+            generators.append(transformer)
+
+    return tuple(generators)
```

Comment:
We should either use a flag `GENERATOR` or change the `OUTPUT_SDTYPES` as I realized that in some (most) of the cases the transformers just override `get_output_sdtypes` directly.